rdrf/rdrf/form_view.py

```diff
@@ -502,9 +524,22 @@ def post(self, request, registry_code, form_id, patient_id, context_id=None):
 
                     sections_to_save.append(section_info)
 
-                    current_data = dyn_patient.load_dynamic_data(self.registry.code, "cdes")
 
-                    form_data = wrap_file_cdes(registry_code, dynamic_data, current_data, multisection=True)
+                    form_data = wrap_file_cdes(registry_code, dynamic_data, current_data, multisection=True, index_map=index_map)
+                    logger.debug("form_data on the form for section %s  = %s" % (s, form_data)) 
+                    try:
```

Comment:
In general, not sure if you want to leave all that detailed logging that has been added, but especially true for this block of code. 
Does it have to stay?